src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -330,7 +346,12 @@ foam.CLASS({
             .end()
             .start()
               .addClass(this.myClass('header-center'))
-              .start({ class: 'foam.nanos.u2.navigation.ApplicationLogoView' })
+              .start(this.Image, {
```

Comment:
@sarthak-marwaha This came from v4.29.  Which do we want?

Reply:
The line being removed should be kept instead